app/assets/images/payment_icons/mobilepay.svg

```diff
@@ -1 +1,16 @@
\ No newline at end of file
+<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" role="img" width="38" height="24" aria-labelledby="pi-mobilepay">
+<title id="pi-mobilepay">Mobilepay</title>
+<path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/>
+<path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/>
+<path fill-rule="evenodd" clip-rule="evenodd" fill="#5A78FF" d="M18.337,5.118c-0.043-0.109-0.119-0.207-0.228-0.294
+	C18,4.758,17.891,4.726,17.782,4.726c-0.054,0-0.135,0.016-0.245,0.049l-4.912,2.04c-0.142,0.043-0.255,0.152-0.343,0.326
+	c-0.064,0.163-0.064,0.321,0,0.474l4.684,11.26c0.065,0.162,0.18,0.282,0.343,0.358c0.164,0.054,0.321,0.049,0.473-0.017
+	l4.896-2.007c0.152-0.076,0.262-0.196,0.327-0.359c0.064-0.163,0.064-0.314,0-0.457l-0.295-0.767h-0.016l-0.784-1.876
+	c-1.326,0.064-2.453,0.358-3.377,0.881l-1.991-4.766c0.054-0.043,0.103-0.087,0.146-0.13c0.022-0.033,0.049-0.065,0.082-0.098
+	c0.022-0.033,0.038-0.055,0.049-0.065l0.032-0.033c0.043-0.054,0.082-0.098,0.114-0.13c0.718-0.707,1.54-1.267,2.464-1.681
+	L18.337,5.118z M19.023,4.84L20.1,7.468c1.426-0.577,2.692-0.789,3.802-0.637l0.963,2.383c0.609,0.13,1.148,0.332,1.615,0.603v4.815
+	c-1.174-0.609-2.436-0.908-3.785-0.898l0.995,2.367c0.142,0.337,0.142,0.69,0,1.061c-0.142,0.337-0.386,0.571-0.734,0.701
+	l-4.896,2.024C17.886,19.963,17.717,20,17.554,20c-0.272,0-0.522-0.076-0.75-0.229c-0.24-0.142-0.408-0.342-0.506-0.604
+	L11.614,7.892c-0.13-0.337-0.125-0.68,0.016-1.028c0.141-0.337,0.386-0.582,0.734-0.734l4.879-2.023
+	c0.359-0.142,0.702-0.142,1.028,0C18.631,4.258,18.881,4.503,19.023,4.84z"/>
+</svg>
```

Comment:
May be able to drop this change in favour of https://github.com/activemerchant/payment_icons/pull/424.